src/components/theme/Comments/Comments.jsx

```diff
@@ -288,205 +173,206 @@ class Comments extends Component {
+        <div id="comment-add-id">
+          <Form
+            onSubmit={onSubmit}
+            onCancel={onEditCancel}
+            submitLabel={intl.formatMessage(messages.comment)}
+            resetAfterSubmit
+            schema={makeFormSchema(intl)}
+          />
+        </div>
+      )}
+      {/* all comments  */}
+      <Comment.Group threaded>
+        {allPrimaryComments.map((item) => commentElement(item))}
+      </Comment.Group>
 
-        {/* load more button */}
-        {this.props.items_total > this.props.items.length && (
-          <Button fluid basic color="blue" onClick={this.loadMoreComments}>
-            <FormattedMessage id="Load more" defaultMessage="Load more..." />
-          </Button>
-        )}
+      {/* load more button */}
+      {items_total > items.length && (
+        <Button fluid basic color="blue" onClick={loadMoreComments}>
+          <FormattedMessage id="Load more" defaultMessage="Load more..." />
+        </Button>
+      )}
 
-        {this.state.replyTo && (
-          <Portal
-            node={
-              document &&
-              document.getElementById(`reply-place-${this.state.replyTo}`)
-            }
-          >
-            <Form
-              onSubmit={this.onSubmit}
-              onCancel={this.onEditCancel}
-              submitLabel={this.props.intl.formatMessage(messages.comment)}
-              resetAfterSubmit
-              schema={makeFormSchema(this.props.intl)}
-            />
-          </Portal>
-        )}
-      </Container>
-    );
-  }
-}
+      {replyTo && (
+        <Portal
+          node={document && document.getElementById(`reply-place-${replyTo}`)}
+        >
+          <Form
+            onSubmit={onSubmit}
+            onCancel={onEditCancel}
+            submitLabel={intl.formatMessage(messages.comment)}
+            resetAfterSubmit
+            schema={makeFormSchema(intl)}
+          />
+        </Portal>
+      )}
+    </Container>
+  );
+};
 
-export default compose(
-  injectIntl,
-  injectLazyLibs(['moment']),
-  connect(
-    (state) => ({
-      items: state.comments.items,
-      next: state.comments.next,
-      items_total: state.comments.items_total,
-      permissions: state.comments.permissions || {},
-      addRequest: state.comments.add,
-      deleteRequest: state.comments.delete,
+Comments.propTypes = {
+  addComment: PropTypes.func.isRequired,
+  deleteComment: PropTypes.func.isRequired,
+  listComments: PropTypes.func.isRequired,
+  listMoreComments: PropTypes.func.isRequired,
+  pathname: PropTypes.string.isRequired,
+  items: PropTypes.arrayOf(
+    PropTypes.shape({
+      author_name: PropTypes.string,
+      creation_date: PropTypes.string,
+      text: PropTypes.shape({
+        data: PropTypes.string,
+        'mime-type': PropTypes.string,
+      }),
+      is_deletable: PropTypes.bool,
+      is_editable: PropTypes.bool,
     }),
-    { addComment, deleteComment, listComments, listMoreComments },
```

Comment:
Please remove proptypes which are not passed up in the parent and are from mapStateToProps as we are now using hooks